README.rst

```diff
@@ -133,66 +143,84 @@ For other build systems
 
 Then, from your source files, include ``xmath/xmath.h``.
 
-Unit Tests
-----------
+Unit Tests and Examples
+-----------------------
+
+This project uses CMake to build the unit test and example applications. Use the steps described above to
+configure and build the unit test and example applications. Both unit test and example projects currently target the
+xCORE.ai explorer board and x86 platforms. All unit tests are currently in the `/test/
+<https://github.com/xmos/lib_xcore_math/tree/master/test/>`_ directory:
+
+* `/test/ <https://github.com/xmos/lib_xcore_math/tree/master/test/>`_ - Unit test projects for ``lib_xcore_math``:
+
+  * `bfp_tests/ <https://github.com/xmos/lib_xcore_math/tree/master/test/bfp_tests/>`_ - BFP unit tests
+  * `dct_tests/ <https://github.com/xmos/lib_xcore_math/tree/master/test/dct_tests/>`_ - DCT unit tests
+  * `filter_tests/ <https://github.com/xmos/lib_xcore_math/tree/master/test/filter_tests/>`_ - filtering unit tests
```

Comment:
`filtering` should be `Filtering` to match the format of the other lines.

Reply:
Done